avocado_vt/plugins/vt_joblock.py

```diff
@@ -28,6 +28,8 @@
 
 from ..test import VirtTest
 
+from six.moves import xrange
```

Comment:
since there is no xrange on python3, better use below import statement to make it easily understandable:
```
from six.moves import xrange as range
```